brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-958 Add object fit option for image block

### Description
Added object fit responsive option for image block 

### Screenshots
![Edit Page ‹ WordPress — WordPress 2022-06-28 at 8 14 05 PM](https://user-images.githubusercontent.com/10000232/176208569-764a2019-8476-4577-a9fc-5af34d910790.jpg)


### Types of changes
New feature (non-breaking change which adds functionality)

### How has this been tested?

- Add an image block 
- Set custom height and width for the image 
- Earlier, only the width of the image was getting applied and the height was set to auto. But now height and width will be applied as set in options. 
- Also set object fit property and check if it is getting applied to the image. 
- Tested responsive values for object fit also. 

### Checklist:
